Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript

```diff
@@ -431,10 +431,16 @@ private function textFormatIndentLineRemoves pPreviousLine, pLine
          
       else
       
-         if "then" is among the words of pPreviousLine and token -1 of pPreviousLine is not "then" and token 1 of pPreviousLine is "if" and token - 1 of pLine is "else" then
+         if "then" is among the words of pPreviousLine and token -1 of pPreviousLine is not "then" \
+               and "else" is among the tokens of pPreviousLine and token 1 of pPreviousLine is "if" and token - 1 of pLine is "else" then
             return item 1 of sTextFormatKeywordMap["else"]
          end if
          
+         if "then" is among the words of pPreviousLine and token -1 of pPreviousLine is not "then" \
+               and token 1 of pPreviousLine is "if" and token - 1 of pLine is "else" then
```

Comment:
@BerndN the space between `-` & `1` here gives me the hebegeebes... not sure if it does what we expect either... `token - 1 of pLine`. Actually same above.

Reply:
I could change it to "last token" or "token -1" if I knew how. Do I add a pull-request to this current one with the changes concerning the same lines as this one? I am using the web interface.

Reply:
If you go to your fork you can choose the branch `patch-4` and then edit the file

Reply:
thanks, that worked, no more hebegeebees :)